### ISSUE TITLE
Incorrect request from WCS Team Engine Test Case for GetCoverage - "gml" namespace identifier missing

### ISSUE THREAD

**Issue by hemant559** [opengeospatial/ets-wcs10#29]
Team Engiine Test Suite (1.13), 
There is wrong a URL for the test case(wcs1-0-0:getcoverage_operations-getcoverage_request-time-post-xml-1). 

```
wcs1-0-0:getcoverage_operations-getcoverage_request-time-post-xml-1
> Request d1e16386_1:
>    Method: post
>    URL: http://122.15.218.129:8081/OGC_WCS/OGCReqResDispatcher?
>    Body: 
> <GetCoverage xmlns="http://www.opengis.net/wcs" xmlns:gml="http://www.opengis.net/gml"  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/wcs http://schemas.opengis.net/wcs/1.0.0/getCoverage.xsd" version="1.0.0" service="WCS" ><sourceCoverage>sikkim</sourceCoverage><domainSubset><temporalSubset>
>         <gml:timePosition>2017-02-10T12:00:00Z</timePosition>
> </temporalSubset></domainSubset>
> <output><format>GeoTIFF</format></output></GetCoverage>
>  
```

here timePostion element's end tag does not have gml namespace identifier.
It should be like
 
`<gml:timePosition>2017-02-10T12:00:00Z</gml:timePosition>`

Due to which our server is throwing exception and team engine test case is getting failed.

**Comment by dstenger** [comment]
Thank you for reporting.

Can you provide a test service we can use to reproduce the reported behaviour?

**Comment by hemant559** [comment]
You can using this test service : http://122.15.218.129:8081/OGC_WCS_Test/OGCReqResDispatcher?&VeRsIoN=1.0.0&SeRvIcE=WCS&ReQuEsT=GetCapabilities

**Comment by keshav-nangare** [comment]
@hemant559 

I have updated the namespace `gml:timePosition`  in above test still test getting failed.

Following is the request and response:

```
Request d19e16386_1:
   Method: post
   URL: http://122.15.218.129:8081/OGC_WCS_Test/OGCReqResDispatcher?
   Body: 
<GetCoverage xmlns="http://www.opengis.net/wcs" xmlns:gml="http://www.opengis.net/gml"  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/wcs http://schemas.opengis.net/wcs/1.0.0/getCoverage.xsd" version="1.0.0" service="WCS" ><sourceCoverage>sikkim</sourceCoverage><domainSubset><temporalSubset>
        <gml:timePosition>2017-02-10T12:00:00Z</gml:timePosition>
</temporalSubset></domainSubset>
<output><format>GeoTIFF</format></output></GetCoverage>
   Response from parser parsers:HeadersParser:
      <headers>
   <header>
      <value>HTTP/1.1 200 OK</value>
   </header>
   <header>
      <name>Server</name>
      <value>Apache-Coyote/1.1</value>
   </header>
   <header>
      <name>Content-Type</name>
      <value>application/vnd.ogc.se_xml</value>
   </header>
   <header>
      <name>Transfer-Encoding</name>
      <value>chunked</value>
   </header>
   <header>
      <name>Date</name>
      <value>Wed, 31 Jan 2018 06:28:02 GMT</value>
   </header>
   <!--Response received in [69] milliseconds--></headers>

Result: Failed
```
Can you please check with your server, is server returned the correct response?

**Comment by papalav** [comment]
@hemant559 , @keshav-nangare 

The modified request given above is correct.
The exception should be because of network/connectivity issues on the hosted test server.
Can it be possible to update the test suite and we will run and confirm it as closed?

**Comment by keshav-nangare** [comment]
Fixed with this commit: d86c6012c7d9aa26ed6705a0559485e17747e1ff [PR: #35]
